<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_load_image_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_load_image_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_select_points">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Measurement Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calculate_results">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>280</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>280</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_results">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>640</width>
      <height>30</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Results will be displayed here</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
